include/dc_silencer.h

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
To be considered: this header probably won't be used outside of `hardware` module; perhaps it would be appropriate to move it out of `include` directory to `src/hardware`?

If you think it should stay here, then it should be added to `include/Makefile.am`.

Reply:
At the time we were having discussions about the DOSBox-X implementing the "hardware accurate" Tandy DC offset and then the subsequent DC-stepping to get it back to center (which produces an audible artifact).

Had our solution on the Tandy side proved to indeed leave a DC offset, then I was planning to use the silencer there as well (and not follow X's approach of recreating an audible hardware defect). Turns out our solution eliminates the DC offset entirely proven with some extensive testing (https://github.com/dosbox-staging/dosbox-staging/pull/457).

But yes - even if we wound up using it in Tandy, the header could still be put in hardware :-)  I'll do that; thanks for the tip!